Makefile

```diff
@@ -16,7 +16,8 @@ RUN_LOGS:=$(LOG_DIR)/$(TIMESTAMP)
 
 LAST_LOGS:=$(shell find logs |grep -E "logs/[^/]*$$" |sort |tail -n 1)
 
-CONDA_ENV_PATH := ./tmp/conda/cgfp
+# CONDA_ENV_PATH := ./tmp/conda/cgfp
```

Comment:
This shouldn't be the project home directory. It needs to go in a dedicated folder or it will dump conda artifacts all over the workspace

Reply:
I was using ./tmp/conda/cgfp but don't care where we put it. Maybe ./conda/cgfp is better?